lacework/resource_lacework_resource_group_gcp.go

```diff
@@ -0,0 +1,200 @@
+package lacework
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"log"
+	"strings"
+
+	"github.com/lacework/go-sdk/api"
+)
+
+func resourceLaceworkResourceGroupGcp() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceLaceworkResourceGroupGcpCreate,
+		Read:   resourceLaceworkResourceGroupGcpRead,
+		Update: resourceLaceworkResourceGroupGcpUpdate,
+		Delete: resourceLaceworkResourceGroupGcpDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: importLaceworkResourceGroup,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The resource group name",
+			},
+			"enabled": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     true,
+				Description: "The state of the resource group",
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "The description of the resource group",
+			},
+			"organization": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The GCP organization",
```

Comment:
Let us make sure to call the need to specify the ID and not the org name.
```suggestion
				Description: "The GCP organization unique identifier",
```

Reply:
or
```suggestion
				Description: "The GCP organization ID",
```